internal/resolvers/oembed/facebook.go

```diff
@@ -0,0 +1,72 @@
+package oembed
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/Chatterino/api/pkg/utils"
+)
+
+var (
+	facebookAppAccessToken string
+)
+
+func loadFacebookCredentials() (appID string, appSecret string, exists bool) {
+	if appID, exists = utils.LookupEnv("CHATTERINO_API_OEMBED_FACEBOOK_APP_ID"); !exists {
+		log.Println("No CHATTERINO_API_OEMBED_FACEBOOK_APP_ID specified, won't do special responses for Facebook or Instagram oEmbed")
+		return
+	}
+
+	if appSecret, exists = utils.LookupEnv("CHATTERINO_API_OEMBED_FACEBOOK_APP_SECRET"); !exists {
```

Comment:
`utils.LookupEnv` already applies environment variable prefix (`CHATTERINO_API_`), it's fine to revert that addition.